pandas-stubs/core/series.pyi

```diff
@@ -59,7 +59,7 @@ class Series(Generic[_DType]):
     @overload
     def __getitem__(self, idx: Union[List[_str], Index[int], Series, slice, str]) -> Series: ...
     @overload
-    def __getitem__(self, idx: int) -> [float]: ...
```

Comment:
It turns out these accidental brackets were what led me to believe my issue was solved. I've got a Series of Series, and I'm getting to a nested series of it via its numerical index. I'd change the return type from float to Series, but that would just break it for whoever set it to float in the first place. The correct solution would be to find a way to stub it so it returns the dtype of that Series, and I'm not sure how to do that.

Reply:
Right, you should just be able to do what you say:

```python
def __getitem__(self, idx: int) -> _DType: ...
```

That's what the dtypes are for. See for example how we do it for numpy: https://github.com/predictive-analytics-lab/data-science-types/blob/master/numpy-stubs/__init__.pyi#L263

Reply:
Thank you! I'll work that out now.